skiffWindowsApp/Skiff Desktop/MainWindow.xaml.cs

```diff
@@ -30,6 +32,59 @@ public partial class MainWindow : Window
 
         private WindowState _prevState;
 
+        #region Dark Theme
+
+        [DllImport("dwmapi.dll", PreserveSig = true)]
+        static extern int DwmSetWindowAttribute(IntPtr hwnd, int attr, ref int attrValue, int attrSize);
+        const int DWMWA_CAPTION_COLOR = 35;
+
+        [DllImport("UXTheme.dll", SetLastError = true, EntryPoint = "#138")]
+        static extern bool ShouldSystemUseDarkMode();
```

Comment:
If we don't want to affect all the CoreWebView2 instances (understandable), then we should minimally look at using `ShouldAppsUseDarkMode` (`#132`) instead.

The ordinal-only nature of this API poses an increased risk of calling the wrong API, so I recommend adding OS version detection to the app as well (e.g. `OperatingSystem.Version.Build >= 22000`).

Reply:
I have come up with a solution that avoids making external calls. It watches the registry values. This is mainly because `ShouldAppsUseDarkMode` is apparently broken on newer Windows versions (I tested myself on Windows 11 and it doesn't return the right value).